internal_configure.bzl

```diff
@@ -2,17 +2,16 @@
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-def _parse_my_own_version_from_module_dot_bazel(module_ctx):
-    lines = module_ctx.read(Label("//:MODULE.bazel")).split("\n")
-    for line in lines:
-        parts = line.split("\"")
-        if parts[0] == "    version = ":
-            return parts[1]
-    fail("Failed to parse my own version from `MODULE.bazel`! " +
-         "This should never happen!")
+_INTEGRITIES = {
+    "2.11.1": "sha256-1HWXjaDNwtQ7c/MJEHhnWdWTqdjuBbG2hG0esWxtLgw=",
+    "2.12.0": "sha256-v48kKr0avNN11RanBnSQ+3Gr15UZooLSK25NGSghhac=",
+    "2.13.1": "sha256-UWMeiJYKiFb5xJcCf1XJ8vkRXK+wjAAFQ5g4oFuhe/w=",
+    "2.13.5": "sha256-seIJxCs6ntdNo+CyWk9M1HjYnV77tI8EsnffQn+vYlI=",
```

Comment:
Cool. Merge `HEAD` to pull in 2.13.6.

Reply:
Will do. One nice thing about using `.get()` instead of `[]` on this below is that if the release isn't in this table it'll just fall back to not supplying the integrity value (so same as current behavior)